BPOR/BPOR.Rms/Controllers/ResearcherController.cs

```diff
@@ -170,11 +171,41 @@ public async Task<IActionResult> SubmitStudy(ResearcherStudyFormViewModel model,
         model.PortfolioSubmissionStatusOptions = context.Submitted.ToList();
         model.OutcomeOfSubmissionOptions = context.SubmissionOutcome.ToList();
 
+
+        ViewData["Title"] = getPageTitle(model.Step);
         ViewData["Step"] = model.Step;
         ViewData["TotalSteps"] = model.TotalSteps;
         return View(model);
     }
 
+    private String getPageTitle(int? step)
+    {
+        if (step == null)
+        {
+            return "Submit study";
```

Comment:
you could just rely on the `default` case here.

Reply:
removed the null check, now just relies on the default of the switch statement.